app/(auth)/layout.tsx

```diff
@@ -1,11 +1,16 @@
+"use client"
+
 import React from 'react';
 import Nav from './nav';
+import { PageTitleProvider, usePageTitle } from './PageTitleProvider';
 
 const DashboardLayout = ({ children }: Readonly<{ children: React.ReactNode }>) => {
+	const { title } = usePageTitle();
+
 	return (
 		<div className="flex flex-col w-screen h-screen max-h-screen bg-white overflow-y-auto">
 			<header className="flex justify-center bg-gray-200 border-b p-3">
```

Comment:
Separating the header component and then importing it into layout.tsx would be a better approach.